applications/workspaces/server/workspaces/service/osbrepository/adapters/figshareadapter.py

```diff
@@ -1,12 +1,13 @@
 from cloudharness import log as logger
-from workspaces.models import RepositoryResourceNode, RepositoryResource, FigshareRepositoryResource
+
+from workspaces.models import FigshareRepositoryResource, RepositoryResource, RepositoryResourceNode
```

Comment:
Same doubts raise for figshare and dandi adapter

Reply:
for future, then we already have a starting point...